src/gss_names.c

```diff
@@ -626,9 +708,104 @@ uint32_t gssntlm_inquire_name(uint32_t *minor_status,
                               gss_OID *MN_mech,
                               gss_buffer_set_t *attrs)
 {
-    return GSS_S_UNAVAILABLE;
+    uint32_t retmin;
+    uint32_t retmaj;
+    uint32_t tmpmin;
```

Comment:
These can go on the same line - `uint32_t retmin, retmaj, tmpmin;`.

Reply:
This is a copy+paste from YOUR code - see gssntlm_sspi_session_key().
Also you don't do such a thing in other places, so I just keep the code consistent.
See other functions in the same file:
```
    uint32_t retmin;
    uint32_t retmaj;

```

Reply:
I am ok with variables on each line, to me it is more readable and easy to change.

Reply:
I prefer to keep codestyle of the same file (separate var per line).
From my side - we have 'single var per definition' as a must, not an option.
So this is more handy for me.